presto-product-tests/conf/docker/singlenode-hdp3/README

```diff
@@ -1,2 +0,0 @@
-HDP 3.1 images (code) + HDP 3.1-like configuration.
-See https://github.com/prestosql/presto/issues/1841 for more information.
```

Comment:
This particular readme wasn't totally useless.

Retain this as a javadoc of `SinglenodeHdp3` class